esp-hal/src/soc/esp32c6/peripherals.rs

```diff
@@ -130,5 +129,10 @@ crate::peripherals! {
         (28, [Input, Output] (0 => SPIHD) (0 => SPIHD))
         (29, [Input, Output] () (0 => SPICLK_MUX))
         (30, [Input, Output] (0 => SPID) (0 => SPID))
+    ],
+    dma_channels: [
+        DMA_CH0: DmaChannel0,
```

Comment:
Do you think we should prefix the GDMA devices with `DMA0_CHX`? The P4 will have two GDMA to contend with so we'll need to distinguish between them somehow